openpuc_scrapers/scrapers/ny.py

```diff
@@ -138,55 +119,7 @@ def process_docket(docket: NYPUCDocketInfo) -> str:
         driver.quit()
 
 
-@workflow
-async def full_scraping_workflow() -> List[List[NYPUCFileData]]:
-    """Main workflow that coordinates all scraping tasks"""
-    # Process all industries in parallel
-    industries = list(range(1, 11))
-    industry_results = list(map(process_industry, industries))
-
-    # Combine results from all industries
-    combined_dockets = combine_dockets(docket_lists=industry_results)
-
-    result_filedata: List[List[NYPUCFileData]] = []
-    for docket in combined_dockets:
-        # Flyte Task that downloads the html
-        html = process_docket(docket)
-        # Flyte Task that extracts the data from the html
-        results = extract_rows(html, docket)
-        result_filedata.append(results)
-        print(f"Processed Doc rows for {docket.docket_id}")
-
-    return result_filedata
-
-
-@workflow
-def test_small_workflow() -> List[NYPUCFileData]:
-    """Main workflow that coordinates all scraping tasks"""
-    docket = NYPUCDocketInfo(
-        docket_id="18-G-0736",
-        matter_type="Complaint",
-        matter_subtype="Formal Non-Consumer Related",
-        title="Complaint and Formal Dispute Resolution Request For Expedited Resolution of East Coast Power & Gas, LLC Regarding Annual Reconciliation Charges of KeySpan Gas East Corporation d/b/a National Grid for January - April 2018",
-        organization="East Coast Power & Gas, LLC",
-        date_filed="12/05/2018",
-        industry_affected="Gas",  # This field wasn't provided in the comments
-    )
-    # Flyte Task that downloads the html
-    html = process_docket(docket)
-    # Flyte Task that extracts the data from the html
-    results = extract_rows(html, docket)
-    print(f"Processed Doc rows for {docket.docket_id}")
-
-    send_castables_to_kessler(results)
-
-    return results
-
-
-@task
-def extract_docket_info(
-    html_content: str, docket_info: NYPUCDocketInfo
-) -> List[NYPUCDocketInfo]:
+def extract_docket_info(intermediate: Dict[str, Any]) -> List[NYPUCDocketInfo]:
```

Comment:
@cappuccinocosmico this also does not pertain to an individual case, but to a caselist... should live on `ScraperNY`?

Reply:
Docstring seems out of date, btw